### PR TITLE
Add Prettier

### PR DESCRIPTION
**Pull request by davidpmccormick**
Just adding the rules for now.

Not sure we should do a total repo cleanup in one PR, or just a bit at a time.

[This VSCode extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) with `"eslint.autoFixOnSave": true` is p neat.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  trailingComma: 'es5',
+  singleQuote: true,
+  jsxSingleQuote: false,
```
Comment (jamesgorrie) [inline .prettierrc.js]: So we're going for double quote?

Comment (davidpmccormick) [reply .prettierrc.js]: double in JSX, single elsewhere?

Comment (davidpmccormick) [reply .prettierrc.js]: Actually, looks like that's the default, so probably not necessary here – will check.

File: common/views/components/EventScheduleItem/EventScheduleItem.js [hunk 2]
```diff
@@ -1,83 +1,150 @@
...
+                  { s: 'HNL4' }
+                )}`}
+              >
+                {event.place.title}
+              </p>
+            )}
 
-            <p className={`${spacing({s: 2}, {margin: ['bottom']})} ${font({s: 'HNL5', m: 'HNL4'})}`} dangerouslySetInnerHTML={{__html: event.promoText}} />
+            <p
+              className={`${spacing({ s: 2 }, { margin: ['bottom'] })} ${font({
```
Comment (jamesgorrie) [inline common/views/components/EventScheduleItem/EventScheduleItem.js]: This typeof stuff will need tidying up - but at least it looks really crap now, so we can do it as and when.

**Pull request merged**